### PR TITLE
Add confirm exit setting

### PR DESCRIPTION
**Pull request by Riztard**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -126,7 +126,8 @@ dependencies {
     // Network client
     implementation("com.squareup.okhttp3:okhttp:${Versions.OKHTTP}")
     implementation("com.squareup.okhttp3:logging-interceptor:${Versions.OKHTTP}")
-    implementation("com.squareup.okio:okio:2.6.0")
+    implementation("com.squareup.okhttp3:okhttp-dnsoverhttps:${Versions.OKHTTP}")
+    implementation("com.squareup.okio:okio:2.7.0")
```
Comment (CarlosEsco) [inline app/build.gradle.kts:130]: why did you update okhttp?

Comment (Riztard) [reply app/build.gradle.kts:130]: accidetally got included XD

Comment (Riztard) [reply app/build.gradle.kts:130]: and i change it to 2.7.0, i think it was 2.7.0 too on ino/tachi :tehe:

File: app/build.gradle.kts [hunk 2]
```diff
@@ -33,7 +33,7 @@ android {
     defaultConfig {
         minSdkVersion(AndroidVersions.MIN_SDK)
         targetSdkVersion(AndroidVersions.TARGET_SDK)
-        applicationId = "eu.kanade.tachiyomi"
+        applicationId = "eu.kanade.tachiyomi.riztard"
```
Comment (AbdullahM0hamed) [inline app/build.gradle.kts:36]: Sneak 100

File: app/src/main/res/layout/manga_grid_item.xml [hunk 3]
```diff
@@ -20,9 +20,9 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_gravity="bottom"
-            android:layout_marginStart="10dp"
-            android:layout_marginTop="10dp"
-            android:layout_marginEnd="10dp"
+            android:layout_marginStart="3dp"
+            android:layout_marginTop="3dp"
+            android:layout_marginEnd="3dp"
```
Comment (arkon) [inline app/src/main/res/layout/manga_grid_item.xml:25]: Don't use arbitrary numbers. They should at least be multiples of 4 to align with the typical Android design grid.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -700,7 +700,7 @@
     <string name="charging">Charging</string>
     <string name="clear">Clear</string>
     <string name="close">Close</string>
-    <string name="collasped">Collapsed</string>
+    <string name="collapsed">Collapsed</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:703]: This will mess up existing translations.

Comment (Riztard) [reply app/src/main/res/values/strings.xml:703]: i mess up okay, i didnt plan to push this, i planned to use it on my own 🤦 :

**Comment by Riztard** [comment]
i mess up okay, i didnt plan to push the recent one, i planned to use it on my own 🤦

**Pull request closed without merging**